src/index.js

```diff
@@ -44,10 +59,21 @@ function green (text) {
 
 function closeServerOnTermination (server) {
```

Comment:
As server is now in the root scope, it's not needed to pass the server instance here.